### PR TITLE
Minor tweaks to wording of Attribution Reporting article

### PR DESCRIPTION
**Pull request by samdutton**
@maudnals Could you check my changes? Feel free to add other tweaks or adjust/remove mine.

### CODE REVIEW

File: site/en/docs/privacy-sandbox/attribution-reporting/index.md [hunk 1]
```diff
@@ -82,11 +82,11 @@ The Attribution Reporting API enables the measurement of two events that are lin
 event on a publisher's website, such as a user viewing or clicking an ad, with a subsequent 
 conversion on an advertiser site.
 
-This API supports click-through conversion attribution measurement (available in the first implementation of this API, origin trial) and view-through attribution measurement ([see public explainer](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting.md)).
+This API supports click-through conversion attribution measurement (available in the first implementation of this API, currently in [origin trial](https://web.dev/conversion-measurement/#browser-support-and-similar-apis)) and view-through attribution measurement ([see public explainer](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting.md)).
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting/index.md:85]:
```suggestion
This API supports click-through conversion attribution measurement (available in the first implementation of this API, currently in [origin trial](/conversion-measurement/#browser-support-and-similar-apis)) and view-through attribution measurement ([see public explainer](https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting.md)).
```

**Review by mihajlija — APPROVED** [review]

File: site/en/docs/privacy-sandbox/attribution-reporting/index.md (hunk 1 above)
Comment (mihajlija) [reply site/en/docs/privacy-sandbox/attribution-reporting/index.md:85]: Preemptively approved, but I noticed a few more web.dev links throughout the post that should be changed to relative scheme.

Comment (samdutton) [reply site/en/docs/privacy-sandbox/attribution-reporting/index.md:85]: Ah — thanks. I'll adjust accordingly.

Comment (mihajlija) [reply site/en/docs/privacy-sandbox/attribution-reporting/index.md]: Ohh, sorry then my bad!

**Comment by maudnals** [comment]
Sorry, just saw this now - LGTM!

**Pull request merged**